registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -249,6 +250,7 @@
     "sha256:511dd068ccd2777af4ffc6247b5d4120a2f2c9653cab6bb1487382ee5ba59513": ["v1.11.0-beta.1"]
     "sha256:8665d717c2b0ddcfc35b08f895c6d3f3e4d5ba3abd1c3ead13394015e012c842": ["v1.11.0-beta.2"]
     "sha256:8a84e4e228183193c9ec73cefb5a99016d7f0797240d7c72a71ea4d6797f2ac9": ["v1.11.0-rc.0"]
+    "sha256:a50b74f61ec806546fdc59ec983f00029f310fafd3f3e97ac21f04806975aab8": ["v1.11.0-rc.1"]
```

Comment:
SHA `a50b74f61ec8` matches clusterctl v1.11.0-rc.1 at the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/clusterctl@sha256:a50b74f61ec806546fdc59ec983f00029f310fafd3f3e97ac21f04806975aab8/details?inv=1&invt=Ab5TGA).